app.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const app = express();
+const { getCategories } = require("./controllers/categories.controllers");
+
+app.use(express.json());
+
+app.get("/api/categories", getCategories);
+
+app.use("/*", (req, res, next) => {
```

Comment:
Sorry this is _really_ picky, but Express does have some default error handling for routes that aren't specified which will provide the exact same 404 Not Found - I would urge you to make your message ever-so-slightly different ie "Route Not Found," that way there's absolutely no chance of a false positive from any tests. 
Actually now I think about it maybe the message key is different... 🤔  I couldn't be certain without checking the docs. Would defo say look into this or be doubly sure by providing a different message.